guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -12,8 +12,17 @@ endif::[]
 ifdef::satellite[]
 * Review Known Issues before you begin an upgrade.
 For more information, see {ReleaseNotesURL}ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues in {ProjectName} {ProjectVersion}].
+* If you run {Project} in a disconnected environment, the following additional prerequisites must be met:
+  - You must obtain and deploy Leapp metadata manually. For more information, see https://access.redhat.com/articles/3664871[Leapp utility metadata in-place upgrades of RHEL for disconnected upgrades].
```

Comment:
If you absolutely must use nested lists, please see https://docs.asciidoctor.org/asciidoc/latest/lists/unordered/#nested-unordered-list

(I think it's ok to do so!)


Also, one sentence per line.